storage/leia.go

```diff
@@ -136,15 +136,16 @@ func (k *kvBackedLeiaStore) handleRestore(config LeiaBackupConfiguration) error
 					return err
 				}
 			}
-			set = make([]refDoc, 0)
 			return nil
 		})
 	}
 
```

Comment:
For pro move, initialize `var set []refDoc` here instead of at line 130 to make sure you don't accidentally use the wrong variable in `writeDocuments`. Now the func shadows the outer `set` var.